_typos.toml

```diff
@@ -0,0 +1,16 @@
+files.extend-exclude = [
+    "target/",
+    "man/*",
+]
+
+[default.extend-identifiers]
+hellow554 = "hellow554"
```

Comment:
What is this?

Reply:
So I just went back and added a bit of documentation on the difference between identifiers and words. Basically typos thinks hellow is a misspelling of hello, but instead of saying hellow is fine, we say that hellow554 is fine, but still complain about hellow in general. (words can't include numbers, but identifiers can)